components/01-atoms/links/_link.scss

```diff
@@ -1,6 +1,9 @@
 // @TODO: These custom property settings seem like they shouldn't need to live
```

Comment:
Per out Tech Debt standards can you create ticket in Jira with these details? Also, add an @link comment here linking it back to the ticket. https://www.notion.so/fourkitchens/Technical-Debt-820b9fda626b4b98bac36f910561767f

Reply:
Done